Source/NSView.m

```diff
@@ -4798,6 +4817,12 @@ - (id) initWithCoder: (NSCoder*)aDecoder
 	  [self didAddSubview: sub];
 	}
       RELEASE(subs);
+
+      // Decode the shadow if this is version 2 or greater...
+      if (version >= 2)
+	{
+	  _shadow = [aDecoder decodeObject];
```

Comment:
Again you probably should retain the decoded object as you release it in the dealloc method.

Reply:
Done.